spec/decorators/review_decorator_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe ReviewDecorator do
+  subject(:decorator) { described_class.new(review) }
+
+  let(:review) { create(:review, user: create(:user)) }
+
+  it '#to_string' do
+    expect(decorator.to_string).to eq(I18n.t('book.review') + " \##{review.id}")
```

Comment:
Honestly, I can't figure out what result we receive after this calculation `I18n.t('book.review') + " \##{review.id}"`
Could you please make this test more readable?